src/bin/encode.rs

```diff
@@ -257,8 +287,16 @@ fn main_aux() {
         quiet
     };
 
-    for source_path in sources {
-        handle_path(&mut options, source_path, PathBuf::new().as_path());
+    if sources.len() == 0 {
```

Comment:
Could you add a comment to explain that we switch to stdin if there are no sources?